csharp/extractor/Semmle.Extraction/Options.cs

```diff
@@ -37,13 +37,18 @@ public abstract class CommonOptions : ICommandLineOptions
         /// <summary>
         /// Whether to extract PDB information.
         /// </summary>
-        public bool PDB { get; private set; } = false;
+        public bool PDB { get; set; } = false;
```

Comment:
Maybe consider to use the HandleFlag method to set PDB instead of making PDB mutable (however, this will impact the CIL flag as well - but it seems to be un-unused).